src/js/stores/AppFormStore.js

```diff
@@ -30,6 +32,7 @@ const transformationRules = {
 const resolveMap = {
   appId: "id",
   cmd: "cmd",
+  container: "container",
```

Comment:
I would use the advantage of this resolve map now and map the fieldIds to it's resolve path.
Get rid of ```container``` and handle the field individually.
```
"dockerImage": "container.docker.image"
```